### PR TITLE
youtube-shorts: refactor into less rules

### PR DESCRIPTION
**Pull request by xvello**
Alternative solution to https://github.com/letsblockit/letsblockit/pull/506: as Shorts are "maturing", the markup to insert them into the interface is consolidating around the `ytd-reel-shelf-renderer` and `ytd-rich-shelf-renderer` elements on desktop and the `ytm-reel-shelf-renderer` on mobile. These are popping up into more and more pages, requiring us to update rules accordingly.

- So far, we have used very targeted rules to reduce the risk of false-positives and the performance cost, but now that there's dedicated elements for this, the risk of false-positive is negligible.
- Another argument for keeping per-page rules would be to allow fine-tuning with params, but that would cause a steep increase in maintenance cost when the vast majority of our users don't care.

These rules match the new shelf in the subscription page, reported in https://github.com/letsblockit/letsblockit/pull/506, and many more instances across the interface. I'd love for @letsblockit/contributors to test them out, especially on pages where I removed rules.

### CODE REVIEW

File: data/filters/templates/youtube-shorts.yaml [hunk 1]
```diff
@@ -17,18 +17,11 @@ template: |
...
-  {{! Suggestions, old layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-compact-video-renderer,ytd-shelf-renderer)
-  {{! Suggestions, new layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer ytd-reel-shelf-renderer
   {{! Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections }}
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] ytd-video-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-item-section-renderer)
-  {{! Channels homepage }}
-  www.youtube.com##ytd-browse[page-subtype="channels"] #contents.ytd-reel-shelf-renderer:upward(ytd-item-section-renderer)
-  {{! Trending section }}
-  www.youtube.com##ytd-browse[page-subtype="trending"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
```
Comment (xvello) [inline data/filters/templates/youtube-shorts.yaml:29]: Was outdated, it's a `ytd-reel-shelf-renderer` now

File: data/filters/templates/youtube-shorts.yaml [hunk 2]
```diff
@@ -17,18 +17,11 @@ template: |
   www.youtube.com##ytd-browse[page-subtype="home"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-rich-item-renderer)
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-grid-video-renderer,ytd-rich-item-renderer)
   www.youtube.com##ytd-search .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
-  {{! Suggestions, old layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-compact-video-renderer,ytd-shelf-renderer)
```
Comment (xvello) [inline data/filters/templates/youtube-shorts.yaml:21]: Removing support for the old-old layout where individual shorts were intermingled with long videos

**Comment by vimode** [comment]
I checked the rules(on desktop) on homepage, subscription page, channel page, search page, single video page(for suggestions?) and trending page, they work fine. Did I miss any?

I don't see Shorts anywhere in those pages 🪓

File: data/filters/templates/youtube-shorts.yaml (hunk 2 above)
Comment (JohnyP36) [reply data/filters/templates/youtube-shorts.yaml:21]: This can indeed be removed.

File: data/filters/templates/youtube-shorts.yaml [hunk 3]
```diff
@@ -17,18 +17,11 @@ template: |
...
   {{! Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections }}
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] ytd-video-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-item-section-renderer)
-  {{! Channels homepage }}
-  www.youtube.com##ytd-browse[page-subtype="channels"] #contents.ytd-reel-shelf-renderer:upward(ytd-item-section-renderer)
-  {{! Trending section }}
-  www.youtube.com##ytd-browse[page-subtype="trending"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
-  {{! Search results }}
-  www.youtube.com##ytd-search #contents ytd-reel-shelf-renderer
+  {{! Wide-band rules to hide generic forms of short shelves across different pages }}
+  www.youtube.com##ytd-rich-shelf-renderer[is-shorts]
```
Comment (JohnyP36) [inline data/filters/templates/youtube-shorts.yaml:23]: This rules also makes the rules regarding `Homepage shelf` outdated (lines 14-15)

afbeelding

And it also makes the rules regarding `Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections` outdated.

afbeelding

**Review by JohnyP36 — CHANGES_REQUESTED** [review]

**Comment by JohnyP36** [comment]
As I first saw a notification on https://github.com/letsblockit/letsblockit/pull/506/files I also responded there

File: data/filters/templates/youtube-shorts.yaml (hunk 3 above)
Comment (xvello) [reply data/filters/templates/youtube-shorts.yaml:23]: Thanks! Will remove these too 🙇

File: data/filters/templates/youtube-shorts.yaml (hunk 1 above)
Comment (JohnyP36) [reply data/filters/templates/youtube-shorts.yaml:29]: The first rule (channel homepage) still works, but is already covered by `ytd-reel-shelf-renderer`
See: 
afbeelding

Only the rule for the trending section is indeed outdated

Thus both can be removed, although the first rule still works, but is not needed anymore.

File: data/filters/templates/youtube-shorts.yaml (hunk 3 above)
Comment (xvello) [reply data/filters/templates/youtube-shorts.yaml:23]: After checking, the list mode on my account still shows individual shorts intertwined in results, with the following markup. I'll keep this rule for the time being, only remove the homepage one.

```html
<!--css-build:shady--><!--css-build:shady--><div id="header" class=" style-scope ytd-item-section-renderer style-scope ytd-item-section-renderer"></div>
<div id="spinner-container" class="style-scope ytd-item-section-renderer">
  <tp-yt-paper-spinner-lite class="style-scope ytd-item-section-renderer" aria-hidden="true"><!--css-build:shady--><div id="spinnerContainer" class="  style-scope tp-yt-paper-spinner-lite"><div class="spinner-layer style-scope tp-yt-paper-spinner-lite"><div class="circle-clipper left style-scope tp-yt-paper-spinner-lite"><div class="circle style-scope tp-yt-paper-spinner-lite"></div></div><div class="circle-clipper right style-scope tp-yt-paper-spinner-lite"><div class="circle style-scope tp-yt-paper-spinner-lite"></div></div></div></div></tp-yt-paper-spinner-lite>
</div>
<div id="contents" class=" style-scope ytd-item-section-renderer style-scope ytd-item-section-renderer"><ytd-shelf-renderer class="style-scope ytd-item-section-renderer" thumbnail-style=""><!--css-build:shady--><!--css-build:shady--><div id="dismissible" class="style-scope ytd-shelf-renderer">
  <div class="grid-subheader style-scope ytd-shelf-renderer">
    <div id="title-container" class="style-scope ytd-shelf-renderer">
      <h2 class="style-scope ytd-shelf-renderer">
        <dom-if class="style-scope ytd-shelf-renderer"><template is="dom-if"></template></dom-if>
        
          <div id="image-container" class="style-scope ytd-shelf-renderer">
            <a class="yt-simple-endpoint style-scope ytd-shelf-renderer" aria-hidden="true" tabindex="-1" title="Carla Lalli Music" href="/@CarlaLalliMusic123">
              <yt-img-shadow id="avatar" class="style-scope ytd-shelf-renderer no-transition" style="background-color: transparent;" loaded=""><!--css-build:shady--><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" src="//yt3.ggpht.com/fdvaRyKBRk6pPa3K-FYlnN_LK3-K9OHenpG9M3vbHUAHNFlZd6HA8-mqC-Kr8HbelS18nkYxIw=s88-c-k-c0x00ffffff-no-rj"></yt-img-shadow>
              <yt-icon id="icon" class="style-scope ytd-shelf-renderer" disable-upgrade="" hidden="">
              </yt-icon>
            </a>
            <div id="title-text" class="style-scope ytd-shelf-renderer">
              <a class="yt-simple-endpoint style-scope ytd-shelf-renderer" href="/@CarlaLalliMusic123">
                <span id="title" class="style-scope ytd-shelf-renderer">Carla Lalli Music</span>
              </a>
              <yt-formatted-string id="title-annotation" class="style-scope ytd-shelf-renderer" is-empty="" hidden=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
            </div>
          </div>
        <dom-if class="style-scope ytd-shelf-renderer"><template is="dom-if"></template></dom-if>
        <ytd-badge-supported-renderer id="featured-badge" class="style-scope ytd-shelf-renderer" disable-upgrade="" hidden="">
        </ytd-badge-supported-renderer>
        <yt-formatted-string id="title-annotation" class="style-scope ytd-shelf-renderer" is-empty=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
        <div id="play-button" class="style-scope ytd-shelf-renderer"></div>
        <div id="sort-filter" class="style-scope ytd-shelf-renderer"></div>
      </h2>
      <div id="spacer" class="style-scope ytd-shelf-renderer"></div>
      <div id="subscribe-button" class="style-scope ytd-shelf-renderer"></div>
      <div id="menu" class="style-scope ytd-shelf-renderer"></div>
    </div>
    <yt-formatted-string id="subtitle" class="can-be-empty style-scope ytd-shelf-renderer" is-empty=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
  </div>
  <div id="contents" class="style-scope ytd-shelf-renderer"><ytd-expanded-shelf-contents-renderer class="style-scope ytd-shelf-renderer"><!--css-build:shady--><!--css-build:shady--><div id="grid-container" class="style-scope ytd-expanded-shelf-contents-renderer"><ytd-video-renderer class="style-scope ytd-expanded-shelf-contents-renderer" bigger-thumbs-style="DEFAULT" lockup="true" inline-title-icon=""><!--css-build:shady--><!--css-build:shady--><div id="dismissible" class="style-scope ytd-video-renderer">
  <ytd-thumbnail use-hovered-property="" class="style-scope ytd-video-renderer" size="medium" loaded=""><!--css-build:shady--><!--css-build:shady--><a id="thumbnail" class="yt-simple-endpoint inline-block style-scope ytd-thumbnail" aria-hidden="true" tabindex="-1" rel="null" href="/shorts/UWfxJoFzgOU">
  <yt-image alt="" ftl-eligible="" notify-on-loaded="" notify-on-unloaded="" class="style-scope ytd-thumbnail"><img alt="" style="background-color: transparent;" class="yt-core-image--fill-parent-height yt-core-image--fill-parent-width yt-core-image yt-core-image--content-mode-scale-aspect-fill yt-core-image--loaded" src="https://i.ytimg.com/vi/UWfxJoFzgOU/hq2.jpg?sqp=-oaymwE2CPYBEIoBSFXyq4qpAygIARUAAIhCGABwAcABBvABAfgBtgiAAoAPigIMCAAQARhyIEsoLTAP&amp;rs=AOn4CLAz6iob_gH8FB392bc_3pqESzJC8g"></yt-image>
  
  <div id="overlays" class="style-scope ytd-thumbnail"><ytd-thumbnail-overlay-time-status-renderer class="style-scope ytd-thumbnail" overlay-style="SHORTS" has-icon=""><!--css-build:shady--><!--css-build:shady--><yt-icon size="16" class="style-scope ytd-thumbnail-overlay-time-status-renderer"><!--css-build:shady--><!--css-build:shady--><yt-icon-shape class="style-scope yt-icon"><icon-shape class="yt-spec-icon-shape"><div style="width: 100%; height: 100%; fill: currentcolor;"><svg width="16" height="16" viewBox="0 0 16 16" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;">
  <path d="M10.65,1C10.65,1,10.65,1,10.65,1c-0.37,0-0.75,0.1-1.09,0.31L4.25,4.46C3.44,4.93,2.96,5.89,3,6.9  C3.05,7.9,3.58,8.77,4.39,9.18c0.02,0.01,0.75,0.35,0.75,0.35l-0.9,0.53c-1.14,0.68-1.58,2.27-0.98,3.55C3.69,14.49,4.5,15,5.35,15  c0.37,0,0.74-0.1,1.09-0.31l5.31-3.15c0.8-0.48,1.29-1.43,1.24-2.45c-0.04-0.99-0.58-1.87-1.39-2.27c-0.02-0.01-0.75-0.35-0.75-0.35  l0.9-0.53c1.14-0.68,1.58-2.27,0.97-3.55C12.31,1.51,11.49,1,10.65,1L10.65,1z"></path>
</svg></div></icon-shape></yt-icon-shape></yt-icon><span id="text" class="style-scope ytd-thumbnail-overlay-time-status-renderer" aria-label="Shorts">
  SHORTS
</span></ytd-thumbnail-overlay-time-status-renderer><ytd-thumbnail-overlay-now-playing-renderer class="style-scope ytd-thumbnail"><!--css-build:shady--><!--css-build:shady--><span id="overlay-text" class="style-scope ytd-thumbnail-overlay-now-playing-renderer">En cours de lecture</span>
<ytd-thumbnail-overlay-equalizer class="style-scope ytd-thumbnail-overlay-now-playing-renderer"><!--css-build:shady--><!--css-build:shady--><svg xmlns="http://www.w3.org/2000/svg" id="equalizer" viewBox="0 0 55 95" class="style-scope ytd-thumbnail-overlay-equalizer">
  <g class="style-scope ytd-thumbnail-overlay-equalizer">
    <rect class="bar style-scope ytd-thumbnail-overlay-equalizer" x="0"></rect>
    <rect class="bar style-scope ytd-thumbnail-overlay-equalizer" x="20"></rect>
    <rect class="bar style-scope ytd-thumbnail-overlay-equalizer" x="40"></rect>
  </g>
</svg>
</ytd-thumbnail-overlay-equalizer>
</ytd-thumbnail-overlay-now-playing-renderer></div>
  <div id="mouseover-overlay" class="style-scope ytd-thumbnail"></div>
  <div id="hover-overlays" class="style-scope ytd-thumbnail"></div>
</a>
</ytd-thumbnail>
  <div class="text-wrapper style-scope ytd-video-renderer">
    <div id="meta" class="style-scope ytd-video-renderer">
      <div id="title-wrapper" class="style-scope ytd-video-renderer">
        <h3 class="title-and-badge style-scope ytd-video-renderer">
          <ytd-badge-supported-renderer collection-truncate="" class="style-scope ytd-video-renderer" disable-upgrade="" hidden="">
          </ytd-badge-supported-renderer>
          <a id="video-title" class="yt-simple-endpoint style-scope ytd-video-renderer" title="The Secret To Perfectly Grilled Salmon #shorts" aria-label="The Secret To Perfectly Grilled Salmon #shorts de Carla Lalli Music il y a 1 jour 35&nbsp;secondes 3 416&nbsp;vues&nbsp;- lire le Short" href="/shorts/UWfxJoFzgOU">
            <yt-icon id="inline-title-icon" class="style-scope ytd-video-renderer" hidden=""><!--css-build:shady--><!--css-build:shady--></yt-icon>
            <yt-formatted-string class="style-scope ytd-video-renderer" aria-label="The Secret To Perfectly Grilled Salmon #shorts de Carla Lalli Music il y a 1 jour 35&nbsp;secondes 3 416&nbsp;vues&nbsp;- lire le Short">The Secret To Perfectly Grilled Salmon #shorts</yt-formatted-string>
          </a>
        </h3>
        <div id="menu" class="style-scope ytd-video-renderer"><ytd-menu-renderer class="style-scope ytd-video-renderer" safe-area=""><!--css-build:shady--><!--css-build:shady--><div id="top-level-buttons-computed" class="top-level-buttons style-scope ytd-menu-renderer"></div><div id="flexible-item-buttons" class="style-scope ytd-menu-renderer"></div><yt-icon-button id="button" class="dropdown-trigger style-scope ytd-menu-renderer" style-target="button"><!--css-build:shady--><!--css-build:shady--><button id="button" class="style-scope yt-icon-button" aria-label="Menu d'actions"><yt-icon class="style-scope ytd-menu-renderer"><!--css-build:shady--><!--css-build:shady--><yt-icon-shape class="style-scope yt-icon"><icon-shape class="yt-spec-icon-shape"><div style="width: 100%; height: 100%; fill: currentcolor;"><svg enable-background="new 0 0 24 24" height="24" viewBox="0 0 24 24" width="24" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;"><path d="M12 16.5c.83 0 1.5.67 1.5 1.5s-.67 1.5-1.5 1.5-1.5-.67-1.5-1.5.67-1.5 1.5-1.5zM10.5 12c0 .83.67 1.5 1.5 1.5s1.5-.67 1.5-1.5-.67-1.5-1.5-1.5-1.5.67-1.5 1.5zm0-6c0 .83.67 1.5 1.5 1.5s1.5-.67 1.5-1.5-.67-1.5-1.5-1.5-1.5.67-1.5 1.5z"></path></svg></div></icon-shape></yt-icon-shape></yt-icon></button><yt-interaction id="interaction" class="circular style-scope yt-icon-button"><!--css-build:shady--><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction></yt-icon-button><yt-button-shape id="button-shape" version="modern" class="style-scope ytd-menu-renderer" disable-upgrade="" hidden=""></yt-button-shape></ytd-menu-renderer></div>
      </div>
      <ytd-video-meta-block class="style-scope ytd-video-renderer" amsterdam-post-mvp=""><!--css-build:shady--><!--css-build:shady-->
<div id="metadata" class="style-scope ytd-video-meta-block">
  <div id="byline-container" class="style-scope ytd-video-meta-block">
    <ytd-channel-name id="channel-name" class=" style-scope ytd-video-meta-block style-scope ytd-video-meta-block"><!--css-build:shady--><!--css-build:shady--><div id="container" class="style-scope ytd-channel-name">
  <div id="text-container" class="style-scope ytd-channel-name">
    <yt-formatted-string id="text" link-inherit-color="" title="" class="style-scope ytd-channel-name complex-string" ellipsis-truncate="" ellipsis-truncate-styling="" has-link-only_=""><a class="yt-simple-endpoint style-scope yt-formatted-string" spellcheck="false" href="/@CarlaLalliMusic123" dir="auto">Carla Lalli Music</a></yt-formatted-string>
  </div>
  <tp-yt-paper-tooltip fit-to-visible-bounds="" class="style-scope ytd-channel-name" role="tooltip" tabindex="-1"><!--css-build:shady--><div id="tooltip" class="hidden style-scope tp-yt-paper-tooltip" style-target="tooltip">
  
    Carla Lalli Music
  
</div>
</tp-yt-paper-tooltip>
</div>
<ytd-badge-supported-renderer class="style-scope ytd-channel-name" system-icons=""><!--css-build:shady--><!--css-build:shady--><div role="status" class="badge badge-style-type-verified style-scope ytd-badge-supported-renderer style-scope ytd-badge-supported-renderer" aria-label="Validé"><yt-icon size="16" class="style-scope ytd-badge-supported-renderer"><!--css-build:shady--><!--css-build:shady--><yt-icon-shape class="style-scope yt-icon"><icon-shape class="yt-spec-icon-shape"><div style="width: 100%; height: 100%; fill: currentcolor;"><svg height="24" viewBox="0 0 24 24" width="24" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;"><path d="M12 2C6.5 2 2 6.5 2 12s4.5 10 10 10 10-4.5 10-10S17.5 2 12 2zM9.8 17.3l-4.2-4.1L7 11.8l2.8 2.7L17 7.4l1.4 1.4-8.6 8.5z"></path></svg></div></icon-shape></yt-icon-shape></yt-icon><span class="style-scope ytd-badge-supported-renderer"></span><tp-yt-paper-tooltip position="top" class="style-scope ytd-badge-supported-renderer" role="tooltip" tabindex="-1"><!--css-build:shady--><div id="tooltip" class="hidden style-scope tp-yt-paper-tooltip" style-target="tooltip">
  Validé
</div>
</tp-yt-paper-tooltip></div><dom-repeat id="repeat" as="badge" class="style-scope ytd-badge-supported-renderer"><template is="dom-repeat"></template></dom-repeat></ytd-badge-supported-renderer>
</ytd-channel-name>
    <div id="separator" class="style-scope ytd-video-meta-block">•</div>
    <yt-formatted-string id="video-info" class="style-scope ytd-video-meta-block" is-empty="" hidden=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
  </div>
  <div id="metadata-line" class="style-scope ytd-video-meta-block">
    
    <ytd-badge-supported-renderer class="inline-metadata-badge style-scope ytd-video-meta-block" system-icons="" hidden=""><!--css-build:shady--><!--css-build:shady--><dom-repeat id="repeat" as="badge" class="style-scope ytd-badge-supported-renderer"><template is="dom-repeat"></template></dom-repeat></ytd-badge-supported-renderer>
    <div id="separator" class="style-scope ytd-video-meta-block" hidden="">•</div>
    
      <span class="inline-metadata-item style-scope ytd-video-meta-block">3,4&nbsp;k&nbsp;vues</span>
    
      <span class="inline-metadata-item style-scope ytd-video-meta-block">il y a 1 jour</span>
    <dom-repeat strip-whitespace="" class="style-scope ytd-video-meta-block"><template is="dom-repeat"></template></dom-repeat>
  </div>
</div>
<div id="additional-metadata-line" class="style-scope ytd-video-meta-block">
  <dom-repeat class="style-scope ytd-video-meta-block"><template is="dom-repeat"></template></dom-repeat>
</div>

</ytd-video-meta-block>
    </div>
    <div id="channel-info" class="style-scope ytd-video-renderer" hidden="">
      <a id="channel-thumbnail" class="style-scope ytd-video-renderer" href="/@CarlaLalliMusic123" aria-label="Accéder à la chaîne">
        <yt-img-shadow width="24" class="style-scope ytd-video-renderer no-transition empty" style="background-color: transparent;"><!--css-build:shady--><!--css-build:shady--><img id="img" draggable="false" class="style-scope yt-img-shadow" alt="" width="24"></yt-img-shadow>
      </a>
      <ytd-channel-name id="channel-name" class="long-byline style-scope ytd-video-renderer" wrap-text="true"><!--css-build:shady--><!--css-build:shady--><div id="container" class="style-scope ytd-channel-name">
  <div id="text-container" class="style-scope ytd-channel-name">
    <yt-formatted-string id="text" link-inherit-color="" title="" class="style-scope ytd-channel-name" has-link-only_=""><a class="yt-simple-endpoint style-scope yt-formatted-string" spellcheck="false" href="/@CarlaLalliMusic123" dir="auto">Carla Lalli Music</a></yt-formatted-string>
  </div>
  <tp-yt-paper-tooltip fit-to-visible-bounds="" class="style-scope ytd-channel-name" role="tooltip" tabindex="-1"><!--css-build:shady--><div id="tooltip" class="hidden style-scope tp-yt-paper-tooltip" style-target="tooltip">
  
    Carla Lalli Music
  
</div>
</tp-yt-paper-tooltip>
</div>
<ytd-badge-supported-renderer class="style-scope ytd-channel-name" system-icons=""><!--css-build:shady--><!--css-build:shady--><div role="status" class="badge badge-style-type-verified style-scope ytd-badge-supported-renderer style-scope ytd-badge-supported-renderer" aria-label="Validé"><yt-icon size="16" class="style-scope ytd-badge-supported-renderer"><!--css-build:shady--><!--css-build:shady--><yt-icon-shape class="style-scope yt-icon"><icon-shape class="yt-spec-icon-shape"><div style="width: 100%; height: 100%; fill: currentcolor;"><svg height="24" viewBox="0 0 24 24" width="24" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;"><path d="M12 2C6.5 2 2 6.5 2 12s4.5 10 10 10 10-4.5 10-10S17.5 2 12 2zM9.8 17.3l-4.2-4.1L7 11.8l2.8 2.7L17 7.4l1.4 1.4-8.6 8.5z"></path></svg></div></icon-shape></yt-icon-shape></yt-icon><span class="style-scope ytd-badge-supported-renderer"></span><tp-yt-paper-tooltip position="top" class="style-scope ytd-badge-supported-renderer" role="tooltip" tabindex="-1"><!--css-build:shady--><div id="tooltip" class="hidden style-scope tp-yt-paper-tooltip" style-target="tooltip">
  Validé
</div>
</tp-yt-paper-tooltip></div><dom-repeat id="repeat" as="badge" class="style-scope ytd-badge-supported-renderer"><template is="dom-repeat"></template></dom-repeat></ytd-badge-supported-renderer>
</ytd-channel-name>
    </div>
    <yt-formatted-string id="description-text" class="style-scope ytd-video-renderer">#carlalallimusic #grilledsalmon #cookinghacks</yt-formatted-string>
    <dom-repeat class="style-scope ytd-video-renderer" hidden=""><template is="dom-repeat"></template></dom-repeat>
    <ytd-badge-supported-renderer id="badges" class="style-scope ytd-video-renderer" disable-upgrade="" hidden="">
    </ytd-badge-supported-renderer>
    <div id="expandable-metadata" class="style-scope ytd-video-renderer"></div>
    <div id="buttons" class="style-scope ytd-video-renderer"></div>
  </div>
</div>
<div id="dismissed" class="style-scope ytd-video-renderer"></div>
<yt-interaction id="interaction" class="extended style-scope ytd-video-renderer"><!--css-build:shady--><!--css-build:shady--><div class="stroke style-scope yt-interaction"></div><div class="fill style-scope yt-interaction"></div></yt-interaction>
</ytd-video-renderer></div>
<yt-formatted-string aria-role="button" class="style-scope ytd-expanded-shelf-contents-renderer" is-empty="" hidden=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
  <a class="yt-simple-endpoint style-scope ytd-expanded-shelf-contents-renderer" hidden="">
  <yt-formatted-string class="style-scope ytd-expanded-shelf-contents-renderer" is-empty=""><!--css-build:shady--><!--css-build:shady--><yt-attributed-string class="style-scope yt-formatted-string"></yt-attributed-string></yt-formatted-string>
</a>
</ytd-expanded-shelf-contents-renderer></div>
</div>
<div id="dismissed" class="style-scope ytd-shelf-renderer"></div>
</ytd-shelf-renderer></div>
<div id="continuations" class="style-scope ytd-item-section-renderer"></div>
```

**Review by JohnyP36 — APPROVED** [review]

File: data/filters/templates/youtube-shorts.yaml [hunk 4]
```diff
@@ -11,24 +11,17 @@ template: |
...
-  {{! Suggestions, old layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-compact-video-renderer,ytd-shelf-renderer)
-  {{! Suggestions, new layout }}
-  www.youtube.com##ytd-watch-next-secondary-results-renderer ytd-reel-shelf-renderer
   {{! Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections }}
   www.youtube.com##ytd-browse[page-subtype="subscriptions"] ytd-video-renderer .ytd-thumbnail[href^="/shorts/"]:upward(ytd-item-section-renderer)
-  {{! Channels homepage }}
-  www.youtube.com##ytd-browse[page-subtype="channels"] #contents.ytd-reel-shelf-renderer:upward(ytd-item-section-renderer)
   {{! Trending section }}
   www.youtube.com##ytd-browse[page-subtype="trending"] .ytd-thumbnail[href^="/shorts/"]:upward(ytd-video-renderer)
```
[the remark this answers is not in the document]
Comment (JohnyP36) [reply data/filters/templates/youtube-shorts.yaml:21]: You are right. The trending Games doesn't have a section but the Shorts are in between the videos like in `https://www.youtube.com/feed/trending?bp=4gIcGhpnYW1pbmdfY29ycHVzX21vc3RfcG9wdWxhcg%3D%3D`.

**Pull request merged**